server/utils/crmFieldFormatter.test.ts

```diff
@@ -0,0 +1,44 @@
+import { formatCurrency, formatDate, formatTime } from './crmFieldFormatter'
+
+describe('CRM Display Field Formatter', () => {
+  describe('formatCurrency', () => {
+    it('should format currency when given whole number', () => {
+      const result = formatCurrency('10000')
+      expect(result).toEqual('£10,000.00')
+    })
+
+    it('should format currency when given fractional number', () => {
+      const result = formatCurrency('9.99')
+      expect(result).toEqual('£9.99')
+    })
+
+    it('should not format currency if not a number', () => {
+      const result = formatCurrency('blah')
+      expect(result).toEqual('blah')
+    })
+  })
+
+  describe('formatDate', () => {
+    it('should format date', () => {
+      const result = formatDate('2024-02-14T00:00:00.000+00:00')
+      expect(result).toEqual('14 02 2024')
```

Comment:
The date format should not retrieve the `0` in day and month
For eg.: 14 2 2024